packages/components/src/progress-bar/index.tsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import type { ForwardedRef } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { forwardRef } from '@wordpress/element';
+import { useInstanceId } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import * as ProgressBarStyled from './styles';
+import type { ProgressBarProps } from './types';
+import type { WordPressComponentProps } from '../ui/context';
+
+// Width of the indicator for the indeterminate progress bar
+export const INDETERMINATE_TRACK_WIDTH = 50;
+
+function UnforwardedProgressBar(
+	props: WordPressComponentProps< ProgressBarProps, 'div', false >,
```

Comment:
```suggestion
	props: WordPressComponentProps< ProgressBarProps, 'progress', false >,
```

I just noticed we aren't forwarding rest props. We should probably do it so consumers can add their own aria labeling (`aria-label`, `aria-labelledby`, etc).

(The `WordPressComponentProps` utility type is basically an intersection of the first argument, and the props of the HTML element specified in the second argument. So we only use it when the rest props are being forwarded.)

Reply:
Makes sense, addressed as part of f0c3be6d7ca283c5e71d7338675cdd20c542113d.